avionics/common/include/rocket.h

```diff
@@ -107,28 +125,30 @@ class CommandReceiver {
         case 0x18:
             break;
             // Not implemented - still uncertain about IMU storage
-        case 0x19:
+        case LATITUDE:
             Radio::sendSingleSensor(sensor_poll_time, 0x19,
                                     rocket_.sensors.gps.getData()[0]);
             break;
-        case 0x1A:
+        case LONGITUDE:
             Radio::sendSingleSensor(sensor_poll_time, 0x1A,
                                     rocket_.sensors.gps.getData()[1]);
             break;
-        case 0x1B:
+        case GPS_ALTITUDE:
             Radio::sendSingleSensor(sensor_poll_time, 0x1B,
                                     rocket_.sensors.gps.getData()[2]);
             break;
-        case 0x1C:
+        case CALCULATED_ALTITUDE:
             Radio::sendSingleSensor(sensor_poll_time, 0x1C,
                                     rocket_.calc.altitude());
             break;
-        case 0x1D:
+        case STATE:
             Radio::sendState(sensor_poll_time, static_cast<uint16_t>(state));
             break;
-        case 0x1E:
-            break; // Voltage sensor not implemented
-        case 0x1F:
+        case VOLTAGE:
+            Radio::sendSingleSensor(sensor_poll_time, 0x1E,
```

Comment:
```suggestion
            Radio::sendSingleSensor(sensor_poll_time, VOLTAGE,
```
Now that you're using the enum, these can be changed too.